# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/DispatcherTrait.h
# Commit: 36b0646e8649
# Full Hash: 36b0646e86498662e770756ad54b1fd648a82184
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1620594
# File Overlap Count: 1
# Description:
#   Bug 1620594 - Part 7: Remove TabGroup and SystemGroup. r=nika,bas
#   
#   TabGroup never really made any difference in which thread something go
#   dispatched to. This was the intended use, but development of TabGroups
#   with abstract main threads never made it that far. The good thing is
# ==============================================================================

diff -r a5ad3b179c92 -r 36b0646e8649 dom/base/DispatcherTrait.h
--- a/dom/base/DispatcherTrait.h	Tue Apr 07 15:17:14 2020 +0000
+++ b/dom/base/DispatcherTrait.h	Tue Apr 07 15:17:47 2020 +0000
@@ -16,12 +16,10 @@
 namespace mozilla {
 class AbstractThread;
 namespace dom {
-class TabGroup;
-
-// This trait should be attached to classes like nsIGlobalObject and Document
-// that have a DocGroup or TabGroup attached to them. The methods here should
-// delegate to the DocGroup or TabGroup. We can't use the Dispatcher class
-// directly because it inherits from nsISupports.
+// This trait should be attached to classes like nsIGlobalObject and
+// Document that have a DocGroup attached to them. The methods here
+// should delegate to the DocGroup. We can't use the
+// Dispatcher class directly because it inherits from nsISupports.
 class DispatcherTrait {
  public:
   // This method may or may not be safe off of the main thread. For Document it